web/src/lib/components/shared-components/side-bar/more-information-albums.svelte

```diff
@@ -19,6 +19,6 @@
   <LoadingSpinner />
 {:then data}
   <div>
-    <p>{data[albumCountType].toLocaleString($locale)} Album{s(data[albumCountType])}</p>
+    <p>{$t('albums_count', { values: { count: data[albumCountType].toLocaleString($locale) } })}</p>
```

Comment:
This parses the localized string as a number and converts it to a string again. That's unnecessary and could cause parsing errors. I suggest passing the number directly here, as well as for `videos_count` and `photos_count`.

```suggestion
    <p>{$t('albums_count', { values: { count: data[albumCountType] })}</p>
```

Reply:
Thought so, however left it like it was before. Fixing...